packages/react-components/src/components/Button/NavButton.jsx

```diff
@@ -0,0 +1,48 @@
+import { forwardRef } from 'react'
+import { func, bool, string, oneOf } from 'prop-types'
+import BaseButton from './BaseButton'
+import theme from '../theme'
+
+const NavButton = forwardRef(
+  ({ disabled, onClick, variant, title, type, isActive, ...props }, ref) => {
+    const activeColor = theme.colors.core.primary
+    const defaultColor = theme.colors.buttons.secondary.borderColor
+
+    // overrides for the base button
+    const buttonDefaultStyles = {
+      borderRadius: 50,
+      color: isActive ? activeColor : defaultColor,
+      borderColor: isActive ? activeColor : defaultColor,
+      borderWidth: isActive ? 2 : 'auto'
+    }
+
+    return (
+      <BaseButton
```

Comment:
I actually think this should be an `a` tag now that I think about it. And instead of `onClick`, we should just pass an `href`. What do you think?

Reply:
Yeah it totally could be, although the style components aren't set up for links right now, only buttons. Should we either:
 - add an attribute for "isLink", and in BaseButton.js return a link instead of a button if it's a link? Or use an "href" flag to ducktype that and say if href make it a link?
 - Add a BaseLink that does the same thing styled correctly?
 
 What do you think?

Reply:
I don't think we should use Button at all for this. I think that's too confusing to outside consumers of this library. 

`secondary` style is close, but its still more rectangular than circular.

We could build in the same file as https://github.com/glifio/modules/blob/primary/packages/react-components/src/components/Link/index.jsx, take some of the properties from the secondary button variant, round the corners some more, and make it an `a` tag